README-PyPi.md

```diff
@@ -1,8 +1,8 @@
 # Qiskit Code Assistant (Beta)
 
-> This experimental feature is only available, as of today, for some IBM Quantum users.
-> If you are not part of that cohort of users, you can still install this extension; however you will not be able to use the assistant.
-> The Qiskit Code Assistant is a preview feature release, subject to change.
+> This experimental feature is only available, as of today, to IBM Quantum premium users.
+> If you are not part IBM Quantum premium plan, you can still install this extension; however you will not be able to use the assistant.
```

Comment:
the grammar here feels off now, it should probably be "part **_of an_** IBM Quantum premium plan"

Reply:
Fixed both comments